po/pl.po

```diff
@@ -4499,19 +4492,16 @@ msgid "_Pay Invoice"
 msgstr "_Zapłać fakturę"
 
 #: gnucash/gnome/gnc-plugin-page-invoice.c:278
-#, fuzzy
 msgid "_Print Bill"
-msgstr "Wydrukuj"
+msgstr "_Drukuj rachunek"
 
 #: gnucash/gnome/gnc-plugin-page-invoice.c:279
-#, fuzzy
 msgid "_Edit Bill"
-msgstr "Edytuj rachunek"
+msgstr "E_dytuj rachunek"
 
 #: gnucash/gnome/gnc-plugin-page-invoice.c:280
-#, fuzzy
 msgid "_Duplicate Bill"
-msgstr "Zduplikuj"
+msgstr "Duplikuj rachunek"
```

Comment:
This is one of 15 places, where the accelerator is missing. The full output from `msgfmt -c --check-accelerators="_" --statistics pl.po` is

> pl.po:3783: msgstr lacks the keyboard accelerator mark '_'
> pl.po:3801: msgstr lacks the keyboard accelerator mark '_'
> pl.po:4504: msgstr lacks the keyboard accelerator mark '_'
> pl.po:5136: msgstr lacks the keyboard accelerator mark '_'
> pl.po:8547: msgstr lacks the keyboard accelerator mark '_'
> pl.po:12320: msgstr lacks the keyboard accelerator mark '_'
> pl.po:12334: msgstr lacks the keyboard accelerator mark '_'
> pl.po:16365: msgstr lacks the keyboard accelerator mark '_'
> pl.po:16945: msgstr lacks the keyboard accelerator mark '_'
> pl.po:17272: msgstr lacks the keyboard accelerator mark '_'
> pl.po:17276: msgstr lacks the keyboard accelerator mark '_'
> pl.po:17448: msgstr lacks the keyboard accelerator mark '_'
> pl.po:20166: msgstr lacks the keyboard accelerator mark '_'
> pl.po:21146: msgstr lacks the keyboard accelerator mark '_'
> pl.po:25425: msgstr lacks the keyboard accelerator mark '_'
> msgfmt: found 15 fatal errors
> 3181 translated messages, 1540 fuzzy translations, 685 untranslated messages.
https://wiki.gnucash.org/wiki/Translation#Check_syntax_and_statistics_of_your_.po_file

Reply:
I know, I know... there are so many works... not for one, two or even three days of work. I will be adding corrections successively. I will let you know when I'm done and you can merge the changes.